src/pages/AddPatient.jsx

```diff
@@ -15,33 +15,29 @@ import { useState } from "react";
 import Toast from "../components/Toast";
 const AddPatient = ({ open, onClose }) => {
   const { register, handleSubmit, reset } = useForm();
-  const [openToast, setOpenToast] = useState(false);
-  const [toastMessage, setToastMessage] = useState("");
-  const [toastSeverity, setToastSeverity] = useState("");
+  const [toastData, setToastData] = useState({
+    openToast: false,
+    toastMessage: "",
+    toastSeverity: ""
+  });
+  const updateToastData = (key, value) => {
+    setToastData((prevData) => ({ ...prevData, [key]: value }));
+  };
   const queryClient = useQueryClient();
-  function handleReset() {
-    reset();
-  }
   const { mutate, isPending } = useMutation({
     mutationFn: addPatient,
-    onError: (error) => {
-      setToastSeverity("error");
-      setToastMessage(`${error.message}`);
-      setOpenToast(true);
-    },
     onSuccess: (data) => {
-      setToastSeverity("success");
-      setToastMessage(data);
-      setOpenToast(true);
-      handleReset();
-      const timeout = setTimeout(() => onClose(), 3000);
-      return () => clearTimeout(timeout);
+      updateToastData("toastSeverity", "success");
```

Comment:
Update all 3 properties of the object a tthe same time:
setToastData({"toastSeverity": "success", "toastMessage" : data., "openToast": true})